### ISSUE TITLE
Error in log(y) %*% diag(p): When using philr transform on filtered dataset

### ISSUE THREAD

**Issue by jolespin** [jsilve24/philr#7]
I'm not sure what is happening.  I made sure to add pseudocounts and that all of the data is numeric.  In this, my Otus are the columns and samples are the rows. 

```R
> packageVersion("phyloseq")
[1] ‘1.22.3’
> packageVersion("philr")
[1] ‘1.7.0’

> df_counts_pseudo = df_counts + 1
> dim(df_counts_pseudo)
[1]  64 246
> all(df_counts_pseudo > 0)
[1] TRUE
> all(apply(df_counts_pseudo, MARGIN=2, FUN =  is.numeric))
[1] TRUE
> tree = read_tree("~/tree.nw")
> class(tree)
[1] "phylo"
> philr(df_counts_pseudo, tree = tree)
Building Sequential Binary Partition from Tree...
Building Contrast Matrix...
Transforming the Data...
Error in log(y) %*% diag(p) : 
  requires numeric/complex matrix/vector arguments
```

When I try the transpose:
```R
> philr(t(df_counts_pseudo), tree = tree)
Building Sequential Binary Partition from Tree...
Error in sbp[colnames(df), ] : subscript out of bounds
```

**Comment by jolespin** [comment]
That ended up working! Thanks.  The `df` argument name got me.  

I ended up doing the following to label the nodes:
```R
tree = makeNodeLabel(tree, method="number", prefix="y")
```

**Is there a way to see which `Otus` are in the numerator and denominator of each `node`?**  Let me know if this warrants a `Close` and reopen of another issue.  

Looking forward to exploring the package!

**Comment by jsilve24** [comment]
Ya I think the command is name.balance (that should be what your looking for). If not let me know. 

Thanks for opening issue. I will close it once I update to a more informative error message. Your the second person this week to mention this. 

Justin

**Comment by jsilve24** [comment]
Closed with 096bc511608e8146ae43176962af71cda6f77c3f
-- Added warning message and auto-convert from data.frame to matrix.
